app/src/main/java/ch/beerpro/presentation/details/DetailsViewModel.java

```diff
@@ -64,4 +68,14 @@ public void toggleLike(Rating rating) {
     public Task<Void> toggleItemInWishlist(String itemId) {
         return wishlistRepository.toggleUserWishlistItem(getCurrentUser().getUid(), itemId);
     }
+
+    public float calcAvgRating(List<Rating> ratings){
+        float sum = 0;
+        if(!ratings.isEmpty()){
+            for(Rating rating : ratings){
+                sum += rating.getRating();
+            }
+        }
+        return sum/ratings.size();
```

Comment:
Is calcAvgRating always called when ratings are changed?
If yes, possible divided by 0 exception:
1. Add rating (size = 1)
2. Remove rating (size = 0)